alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetMonitorAction.kt

```diff
@@ -131,7 +131,6 @@ class TransportGetMonitorAction @Inject constructor(
                                         response.version,
                                         response.seqNo,
                                         response.primaryTerm,
-                                        RestStatus.OK,
```

Comment:
What happened with this line

Reply:
See last bullet point in https://github.com/opensearch-project/common-utils/pull/566:

> removed the status field from GetMonitorResponse. This is so we can change the response from extending ActionResponse and ToXContentObject explicitly, and changing to BaseResponse, which has an override for getStatus(). Note the BaseResponse extends the same ActionResponse and ToXContentObject. This follows the same approach as done previously - see https://github.com/opensearch-project/alerting/pull/556/files as an example.